grid-proxy/tests/queries/node_test.go

```diff
@@ -157,7 +157,7 @@ func TestNode(t *testing.T) {
 		assert.NoError(t, err)
 
 		for _, node := range nodes {
-			assert.Equal(t, node.HasGPU, hasGPU, "has_gpu filter did not work")
+			assert.Equal(t, node.NumGPUs, 1, "has_gpu filter did not work")
```

Comment:
i think the message needs rephrasing, it's not has_gpu anymore

Reply:
At this point it still is, will change when we settle on the new filters